.gitignore

```diff
@@ -1,3 +1,6 @@
 .DS_Store
 public/
 resources/
+
+# Temporary lock file while building
+/.hugo_build.lock
```

Comment:
Are you sure that `/` is right here? Elsewhere I see it without ...

Reply:
Take a look at 

https://github.com/github/gitignore/blob/4488915eec0b3a45b5c63ead28f286819c0917de/community/Golang/Hugo.gitignore
Or 
https://github.com/toptal/gitignore/blob/2d56fdde1e84eaeb6688cc3536afdfd3c54c6016/templates/Hugo.gitignore


In the gitignore documentation, the first bullet in the section examples linked below, explain that it is to match only at the root, not inside subdirectories. 
https://git-scm.com/docs/gitignore/2.34.0#_examples

I could retry and make a test that a file called `.hugo_build.lock` stored in the static folder (if by some random chance we would want to make a download named like that), to make sure it isn't ignored.

Reply:
Ok, I see. Fine for me.